docs/docs/dashboard.md

```diff
@@ -103,6 +129,18 @@ Folders can be searched by clicking on the search icon on the left drawer of the
 
 </div>
 
+### Slug for folders
+
+The slug for folders is automatically generated based on the name of the folder. The slug for folders can be used to access the folder directly from the URL. 
```

Comment:
The folder's URL slug is generated automatically from its name, providing direct access to the folder using the slug.
@withshubh please change it to this.